### ISSUE TITLE
Issue - Cannot drill into Flow/App details from Power BI

### ISSUE THREAD

**Issue by breakpoint7** [microsoft/powerapps-tools#314]
After setting up the Power BI Dashboard, I am getting details on flows and apps in my environment, however, when I try to drill through (Flow Detail), I always get an error saying "Sorry, we didn't find that app".  The error detail will list the GUID … could not be found in the tenant...

In the flow detail screen, it lists the correct connectors and actions for the flow, so it seems it's able to pull the right data.  I'm looking for the ability to Manage the Flow and take ownership or delete the flow as discussed in https://docs.microsoft.com/en-us/power-platform/guidance/coe/power-bi-govern (Flow Archive)
![flowdetail_drill](https://user-images.githubusercontent.com/26799308/83426108-a7c4cc80-a3fc-11ea-8de8-b9a1381ceebc.png)
![flowdetails](https://user-images.githubusercontent.com/26799308/83425821-3d139100-a3fc-11ea-8adb-4e68b529662f.png)

**Comment by JeneferM-MSFT** [comment]
Did you do the work to install the apps as described here? [Optional: configure embedded apps in the coe dashboard](https://docs.microsoft.com/power-platform/guidance/coe/setup-powerbi#optional-configure-embedded-apps-in-the-coe-dashboard)

**Comment by breakpoint7** [comment]
Thank you.  That did it.  Might consider adding a note in the docs that this is a required setup component to enable managing flows and apps via the dashboard.  I skipped over this as optional - not realizing what the dependency meant.  It was clear reading back through it, but there enough manual setup involved initially, it's easy to skip the optional stuff.

Also took me a couple passes before I realized that in order to get all the features discussed in - Govern with the CoE Power BI dashboard, I needed to install the *advanced* Power BI dashboard.  I initially just installed the Dashboard from AppSource which seems to add the basic one.  Maybe I missed that in the docs, but it wasn't clear that there were a couple versions of this and the advanced one is in the COE Starter Kit download.

Appreciate your help.
